### ISSUE TITLE
dbNSFP plugin producing empty columns in vep 88.8

### ISSUE THREAD

**Issue by TMiguelT** [Ensembl/ensembl-vep#55]
I've just upgraded to the new VEP from the old Ensembl Tools one, and it seems that after the upgrade, the dbNSFP columns (e.g. `CADD_phred | CADD_raw | CADD_raw_rankscore | COSMIC_CNT | COSMIC_ID | ESP6500_AA_AF | ESP6500_EA_AF`) are being created, but not populated. 

I used the following command: 
```bash
vep
--allele_number
--assembly GRCh37
--cache
--canonical
--check_existing
--dir /hsm/VR0320/shared/production/2.5.0/data/vep_cache
--dir_plugins /hsm/VR0320/shared/production/2.5.0/tools/vep_plugins
--fasta /hsm/VR0320/shared/production/2.5.0/data/vep_cache/homo_sapiens_refseq/*/Homo_sapiens.*.dna.primary_assembly.fa.gz
--force_overwrite
--hgvs
-i variants/NA12878.individual.genotype.soi.vcf
--af_1kg
--af_esp
--af
-o variants/NA12878.individual.genotype.soi.vep.vcf
--offline
--plugin Condel,/hsm/VR0320/shared/production/2.5.0/tools/vep_plugins/config/Condel/config,s 
--plugin dbNSFP,/hsm/VR0320/shared/production/2.5.0/data/dbnsfp/dbNSFP.gz,SIFT_pred,Polyphen2_HDIV_score,Polyphen2_HDIV_rankscore,Polyphen2_HDIV_pred,Polyphen2_HVAR_score,Polyphen2_HVAR_rankscore,Polyphen2_HVAR_pred,LRT_score,LRT_converted_rankscore,LRT_pred,MutationTaster_score,MutationTaster_converted_rankscore,MutationTaster_pred,MutationAssessor_score,MutationAssessor_rankscore,MutationAssessor_pred,FATHMM_score,FATHMM_rankscore,FATHMM_pred,MetaSVM_score,MetaSVM_rankscore,MetaSVM_pred,MetaLR_score,MetaLR_rankscore,MetaLR_pred,Reliability_index,VEST3_score,VEST3_rankscore,PROVEAN_score,PROVEAN_converted_rankscore,PROVEAN_pred,CADD_raw,CADD_raw_rankscore,CADD_phred,GERP++_NR,GERP++_RS,GERP++_RS_rankscore,phyloP46way_primate,phyloP46way_primate_rankscore,phyloP46way_placental,phyloP46way_placental_rankscore,phyloP100way_vertebrate,phyloP100way_vertebrate_rankscore,phastCons46way_primate,phastCons46way_primate_rankscore,phastCons46way_placental,phastCons46way_placental_rankscore,phastCons100way_vertebrate,phastCons100way_vertebrate_rankscore,SiPhy_29way_pi,SiPhy_29way_logOdds,SiPhy_29way_logOdds_rankscore,LRT_Omega,UniSNP_ids,ESP6500_AA_AF,ESP6500_EA_AF,clinvar_rs,clinvar_clnsig,clinvar_trait,COSMIC_ID,COSMIC_CNT
--plugin Grantham
--polyphen b
--protein
--pubmed
--refseq
--sift b
-species homo_sapiens
--symbol
--vcf
--vcf_info_field ANN
--verbose;
```

This is with vep `88.8`, and commid `3be3889` of the vep_plugins git repository.

Is there any reason this might be happening?

**Comment by willmclaren** [comment]
Hello,

You might like to try updating your checkout of the VEP_plugins repo to use either the `dev` or `release/89` branches.

I have tested a simpler command to check, using VEP's included example file for GRCh37 and dbNSFP 2.9.2, and I see results for all of the fields, including those you cite as being missing:

```
$ ./vep -i examples/homo_sapiens_GRCh37.vcf -force -cache -port 3337 -plugin dbNSFP,${HOME}/gpfs/dbNSFP/2.9.2/dbNSFP2.9.2.txt.gz,SIFT_pred,Polyphen2_HDIV_score,Polyphen2_HDIV_rankscore,Polyphen2_HDIV_pred,Polyphen2_HVAR_score,Polyphen2_HVAR_rankscore,Polyphen2_HVAR_pred,LRT_score,LRT_converted_rankscore,LRT_pred,MutationTaster_score,MutationTaster_converted_rankscore,MutationTaster_pred,MutationAssessor_score,MutationAssessor_rankscore,MutationAssessor_pred,FATHMM_score,FATHMM_rankscore,FATHMM_pred,MetaSVM_score,MetaSVM_rankscore,MetaSVM_pred,MetaLR_score,MetaLR_rankscore,MetaLR_pred,Reliability_index,VEST3_score,VEST3_rankscore,PROVEAN_score,PROVEAN_converted_rankscore,PROVEAN_pred,CADD_raw,CADD_raw_rankscore,CADD_phred,GERP++_NR,GERP++_RS,GERP++_RS_rankscore,phyloP46way_primate,phyloP46way_primate_rankscore,phyloP46way_placental,phyloP46way_placental_rankscore,phyloP100way_vertebrate,phyloP100way_vertebrate_rankscore,phastCons46way_primate,phastCons46way_primate_rankscore,phastCons46way_placental,phastCons46way_placental_rankscore,phastCons100way_vertebrate,phastCons100way_vertebrate_rankscore,SiPhy_29way_pi,SiPhy_29way_logOdds,SiPhy_29way_logOdds_rankscore,LRT_Omega,UniSNP_ids,ESP6500_AA_AF,ESP6500_EA_AF,clinvar_rs,clinvar_clnsig,clinvar_trait,COSMIC_ID,COSMIC_CNT
$ grep -v "#" variant_effect_output.txt | head -n1
rs116645811     21:26960070     A       ENSG00000154719 ENST00000307301 Transcript      missense_variant        1043    1001    334     T/M     aCg/aTg -       IMPACT=MODERATE;STRAND=-1;CADD_phred=8.543;CADD_raw=0.656999;CADD_raw_rankscore=0.16808;ESP6500_AA_AF=2.27E-4;ESP6500_EA_AF=0.0;FATHMM_pred=T;FATHMM_rankscore=0.43294;FATHMM_score=0.95;GERP++_NR=2.76;GERP++_RS=0.874;GERP++_RS_rankscore=0.19124;LRT_Omega=1.096220;LRT_converted_rankscore=0.07232;LRT_pred=U;LRT_score=0.862718;MetaLR_pred=T;MetaLR_rankscore=0.22820;MetaLR_score=0.0551;MetaSVM_pred=T;MetaSVM_rankscore=0.21075;MetaSVM_score=-1.0256;MutationTaster_converted_rankscore=0.09067;MutationTaster_pred=N,N;MutationTaster_score=1,1;PROVEAN_converted_rankscore=0.31532;PROVEAN_pred=N;PROVEAN_score=-1.24;Polyphen2_HDIV_pred=P;Polyphen2_HDIV_rankscore=0.46020;Polyphen2_HDIV_score=0.871;Polyphen2_HVAR_pred=B;Polyphen2_HVAR_rankscore=0.29440;Polyphen2_HVAR_score=0.102;Reliability_index=9;SIFT_pred=D;SiPhy_29way_logOdds=4.1266;SiPhy_29way_logOdds_rankscore=0.10129;SiPhy_29way_pi=0.1405:0.2416:0.6178:0.0;VEST3_rankscore=0.12795;VEST3_score=0.114;phastCons100way_vertebrate=0.000000;phastCons100way_vertebrate_rankscore=0.05858;phastCons46way_placental=0.000000;phastCons46way_placental_rankscore=0.03702;phastCons46way_primate=0.000000;phastCons46way_primate_rankscore=0.00434;phyloP100way_vertebrate=0.390000;phyloP100way_vertebrate_rankscore=0.20768;phyloP46way_placental=0.226000;phyloP46way_placental_rankscore=0.20979;phyloP46way_primate=-0.176000;phyloP46way_primate_rankscore=0.13171
```

**Comment by TMiguelT** [comment]
Yep! I update my commit version of the vep plugins repository to `4b5d3b1` (which I'm guessing is only on the `release/89` branch) and it all now works.
